tests/PortingAssistant.Client.IntegrationTests/RunPortingCorrectnessWithDotNetFramework.cs

```diff
@@ -45,12 +45,6 @@ public override void OneTimeSetUp()
             }
         }
 
-        [OneTimeTearDown]
```

Comment:
Oh any reason we are getting rid of the tear down function?

Reply:
This test class [inherits](https://github.com/aws/porting-assistant-dotnet-client/blob/b40f1a24513347a737ed23f2cc92a2327b93bd7e/tests/PortingAssistant.Client.IntegrationTests/RunPortingCorrectnessWithDotNetFramework.cs#L10) `CorrectnessTestBase` which [has the cleanup method](https://github.com/aws/porting-assistant-dotnet-client/blob/b40f1a24513347a737ed23f2cc92a2327b93bd7e/tests/PortingAssistant.Client.IntegrationTests/CorrectnessTestBase.cs#L29).